lib/parser/openmp-grammar.h

```diff
@@ -181,8 +181,8 @@ TYPE_PARSER("ALIGNED" >>
                     parenthesized(scalarIntExpr))) ||
     "DIST_SCHEDULE" >> construct<OmpClause>(construct<OmpClause::DistSchedule>(
                            parenthesized("STATIC ," >> scalarIntExpr))) ||
-    "FINAL" >> construct<OmpClause>(
-                   construct<OmpClause::Final>(parenthesized(scalarIntExpr))) ||
```

Comment:
Is this causing a bug? The spec says `scalar-expression`

Reply:
You must be looking at the C/C++ syntax...

Reply:
Ya you're right.